src/engine/ruleset/decoders/sonicwall/firewall-general.yml

```diff
@@ -0,0 +1,63 @@
+---
+name: decoder/firewall-general/0
```

Comment:
Question (naming): rename decoder/firewall-general/0 to decoder/sonicwall-firewall-general/0?

Reply:
I think that since the decoder is inside a folder called sonicwall it becomes a bit redundant, what do you think?

Reply:
Yes, the problem could appear in the environment since only the name of the decoder is specified there. Check for example the decoder/syscollector-*

Reply:
Makes sense. I think that if we follow the current development line, the user manual specified to put typo/name/version. I think that a very long name would end up spoiling it and a short one would remain unfinished. I don't know what to say. what do you think @cborla ?